src/hub.js

```diff
@@ -531,8 +531,8 @@ export async function updateEnvironmentForHub(hub, entryManager) {
   }
 }
 
-export async function updateUIForHub(hub, hubChannel) {
-  remountUI({ hub, entryDisallowed: !hubChannel.canEnterRoom(hub) });
+export async function updateUIForHub(hub, hubChannel, showNewLoaderRefreshPrompt = false) {
+  remountUI({ hub, entryDisallowed: !hubChannel.canEnterRoom(hub), showNewLoaderRefreshPrompt });
```

Comment:
Honestly I don't rellay like to add `shoeNewLoaderRefreshPrompt` param here. Any suggestion for simpler approach is very welcome.

Reply:
Yea I see what you are saying. Since we don't have a global state management tool ( that I know of here ie redux)  this might be what you have to do...

One thing you could do to make it better though is to broaden the name a little more.. instead of making it specific to showing a new loader you could just make it a boolean called pageWillRefresh so if you need it for another reason in the future you could use it.

I might also change the loader references from "new", new is only new until it's not anymore, and then the newnewloader comes :)

Reply:
Yeah, I started to feel like renaming "New Loader". I want to merge this PR as is for now and make another PR for renaming.